mindjunkies/forums/models.py

```
@@ -0,0 +1,85 @@
+    slug = models.SlugField(max_length=255, unique=True, blank=True)
+    content = models.TextField()
+    author = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, related_name='forum_topics')
+    course = models.ForeignKey(Course, on_delete=models.CASCADE, related_name='forum_topics')
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    # Simple reaction counters
+    like_count = models.PositiveIntegerField(default=0)
+
+    class Meta:
+        ordering = ['-created_at']
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)